src/components/Person/Person.jsx

```diff
@@ -1 +1,36 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+function getMarriageStatus(person) {
+  return person.sex === 'm'
+    ? `${person.partnerName} is my wife`
+    : `${person.partnerName} is my husband`;
+}
+
+export const Person = ({ person }) => {
+  const { name, age, isMarried } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        My name is
+        {' '}
+        {name}
```

Comment:
btw, we can simplify it 
```suggestion
        {`My name is ${name}`}
```